CMakeLists.txt

```diff
@@ -168,6 +168,12 @@ else(CCTAG_WITH_CUDA)
   message( STATUS "Building without CUDA" )
 endif(CCTAG_WITH_CUDA)
 
+if( ( CUDA_VERSION VERSION_EQUAL "9.0" ) OR ( CUDA_VERSION VERSION_GREATER "9.0") )
+  set(HAVE_SHFL_DOWN_SYNC   1)
```

Comment:
`CCTAG_HAVE_SHFL_DOWN_SYNC` to be consistent with the other cmake variables

Reply:
OK, done.